test/transpile/module-resolution.spec.ts

```diff
@@ -272,6 +273,49 @@ describe("module resolution project with dependencies built by tstl library mode
     });
 });
 
+describe("module resolution project with dependencies built by tstl library mode and has exports field", () => {
+    const projectPath = path.resolve(__dirname, "module-resolution", "project-with-tstl-library-has-exports-field");
+    const appPath = path.join(projectPath, "app");
+
+    // First compile dependencies into node_modules. NOTE: Actually writing to disk, very slow
+    tstl.transpileProject(path.join(projectPath, "dependency1-ts", "tsconfig.json"));
+
+    // Install dependencies. This will create node_modules folder with dependency1-ts in it.
+    execSync("npm i", { cwd: appPath });
```

Comment:
I don't want to run npm during tests for all kind of reasons (dependency, security, flakyness etc). Please just put dependency1 in node_modules beforehand and compile there, or commit a symlink or something like that.